### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by maggiejgray**
created basic test suite for app

### CODE REVIEW

File: src/js/components/container/app.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentProduct: 1,
+      data: []
+    };
+  }
+
+  componentDidMount() {
```
Comment (drewdistefano) [inline src/js/components/container/app.jsx:15]: What if you moved lines 16-21 into their own method that gets invoked here? Modularizes your code a bit and gives you a function to use if you ever need to fetch the data again.

File: src/js/components/presentational/ImageList.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import React from "react";
+// import PropTypes from "prop-types";
+
+// add functionality to change image on click/hover
+
+const ImageList = (props) => {
```
Comment (drewdistefano) [inline src/js/components/presentational/ImageList.jsx:6]: I did some Googling and found a library called ImageSwipe that might be helpful here.

Comment (drewdistefano) [reply src/js/components/presentational/ImageList.jsx:6]: Link: https://photoswipe.com/

File: server/server.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+// const env = require('dotenv').config(); <<<< NEED TO IMPLEMENT THIS
+
+const port = process.env.PORT || 3003;
+
+const app = express();
+
+app.use(express.static(path.join(__dirname, '../dist')));
+
+app.get('/product_info', (req, res) => {
+  db.query(`SELECT * FROM products WHERE sku = ${Math.floor(Math.random() * 50)}`, (err, response) => {
```
Comment (drewdistefano) [inline server/server.js:13]: You can modularize this a bit by wrapping the query in a helper function and invoking it here.

File: server/server.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    } else{
+      console.log('response in server from db:', response);
+      res.send(response.rows[0]);      
+    }
+  })
+})
+
+
+app.listen(port, () => {
+  console.log(`APP IS LISTENING TO SERVE YOUR NEEDS at http://localhost:${port}`)
```
Comment (drewdistefano) [inline server/server.js:26]: lol

**Review by drewdistefano — APPROVED** [review]

File: src/js/components/container/app.jsx (hunk 1 above)
Comment (maggiejgray) [reply src/js/components/container/app.jsx:15]: Yeah, good idea. I was thinking I should do that after reviewing your code. I will refactor and incorporate db helper functions.

File: src/js/components/presentational/ImageList.jsx (hunk 2 above)
Comment (maggiejgray) [reply src/js/components/presentational/ImageList.jsx:6]: I did see that one, I am going to look into it more.

**Pull request merged**